modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/MultiSelectTag.java

```diff
@@ -46,7 +46,8 @@ public int doStartTag() throws JspException {
 
 		setCloseButtonAriaLabel(
 			LanguageUtil.get(
-				TagResourceBundleUtil.getResourceBundle(pageContext), "close"));
+				TagResourceBundleUtil.getResourceBundle(pageContext),
```

Comment:
You're localizing it in the setter (that should be in the getter), no need to do it here too.

Reply:
Hey @carloslancha I've made all the changes: [9529806](https://github.com/liferay-frontend/liferay-portal/pull/528/commits/95298063acd88bd6bfdd4b15542820c2c7ba23b6), but they didn't make it work. I still get `remove-x` in the hydrated variant.

Reply:
You need to pass as prop the result of `getCloseButtonAriaLabel()` (where it's being localized), not `_closeButtonAriaLabel`

Reply:
That's probably why I was doing it in the setter initially, it just made sense in my head. Lemme try using the getter like you said.

Reply:
It doesn't work, still, here's the change to using getter: [8ee5907](https://github.com/liferay-frontend/liferay-portal/pull/528/commits/8ee5907811b9f9d57203eea85b1543f41bce7a29). I tried a couple variants, just to brute-force, but nope. Is it maybe a problem with the language key?

Reply:
What are you receiving in the component?

Reply:
`remove-x`, all of those buttons get rendered with a `aria-label="remove-x"` and logging `closeButtonAriaLabel` inside MultiSelect.js logs `remove-x`. As we established, it should be translated at that point but it's not

Reply:
Is it maybe because of this line in `doStartTag`->  `setCloseButtonAriaLabel("remove-x");` setting it and locking it at that value somehow?

Reply:
There's a problem getting the key from the module's `Language.properties`. @jbalsas could you help here?

Btw @kresimir-coko `clearAllTitle` and `closeButtonAriaLabel` should be set in `doStartTag` only if they're null.